### PR TITLE
frontend avatar and resume file size check

### LINKED ISSUE #335: Validate sizes of resumes and avatars before uploading

### PR DESCRIPTION
**Pull request by maggiehuiliyao**
closes #335

### CODE REVIEW

**Comment by maggiehuiliyao** [comment]
I'm not sure if the limit is still 5 MiB so let me know if I should change that. I also wasn't sure how to figure out how to edit the status of the avatar upload button to return back to normal so my solution was moving the timeout stuff if that makes sense. So it works supposedly but just it will show the loading symbol even though it isn't really doing anything.

**Comment by n8kim1** [comment]
(putting "frontend" in title just to point that out explicitly -- we don't have backend check yet anyways oops)

**Comment by n8kim1** [comment]
(will push off reviewing this til weds or smth -- i gotta focus on tour stuff)

**Comment by n8kim1** [comment]
(will push off reveiw til after first-tour i think)

File: frontend/src/components/avatarUpload.js [hunk 1]
```diff
@@ -28,10 +28,10 @@ class AvatarUpload extends Component {
...
         this.setState({ status: "failure" });
       }
-      setTimeout(() => {
-        this.setState({ status: "waiting" });
-      }, 2000);
     });
+    setTimeout(() => {
+      this.setState({ status: "waiting" });
+    }, 2000);
   }
```
Comment (n8kim1) [inline frontend/src/components/avatarUpload.js:35]: could you revert this change? realizing it'll make legit things break. I'll think abt what else you could do to mitigate...

**Comment by n8kim1** [comment]
looks good! might be worth refactoring later but not necessary now.

**Review by n8kim1 — APPROVED** [review]

**Pull request merged**